server/poll/transform.go

```diff
@@ -64,10 +72,12 @@ var (
 func (p *Poll) ToPostActions(bundle *utils.Bundle, pluginID, authorName string) []*model.SlackAttachment {
 	localizer := bundle.GetServerLocalizer()
 	numberOfVotes := 0
+	voters := mapset.NewSet[string]()
```

Comment:
I'm not to keen on using a new library. Can we use a `map[string]struct{}` instead and take its `len` later?

Reply:
It's good way, thank you for catching.